Tests/SonarScanner.MSBuild.PreProcessor.Test/JreCaching/JreCacheTests.cs

```diff
@@ -59,14 +62,11 @@ public void CacheDirectoryIsCreated()
     {
         var home = @"C:\Users\user\.sonar";
         var cache = Path.Combine(home, "cache");
```

Comment:
Extract home, cache, sha and file to fields to simplify tests+de-duplicate.

On the ones you need to use different values, just do it in place.